doc/source/NewSiteConfigs.rst

```diff
@@ -745,23 +756,35 @@ With all of that in mind, the following instructions were used on an Amazon Web
          zlib-api: [zlib]
          blas: [nvhpc]
        compiler:
-       - nvhpc@24.3
+       - nvhpc@24.9
      nvhpc:
        externals:
-       - spec: nvhpc@24.3 %nvhpc
+       - spec: nvhpc@24.9 %nvhpc
          modules:
-         - nvhpc/24.3
+         - nvhpc/24.9
        buildable: false
      python:
        buildable: false
        require:
-       - '@3.10.12'
+       - '@3.12.3'
      curl:
        buildable: false
      cmake:
        buildable: false
      pkg-config:
        buildable: false
+     libbsd:
+       buildable: false
+     xz:
+       buildable: false
+
+Then, add the following code block to ``envs/jedi-mpas-nvidia-env/site/packages.yaml`` (because ``spack external find`` doesn't appear to work for this library):
```

Comment:
I have literally no idea what I'm doing inside spack-stack, so if the experts have an idea of a better way to do this, please suggest it!

Reply:
My guess is that `libbsd` is coming is as a dependency of `expat`. I recently changed the `expat` requirements in `config/common/packages.yaml` to disable the `libbsd` variant. Maybe this manual change isn't needed anymore.

Reply:
I can look into this.